File: auth/utils/encoder.js






const payloads = "qwertyuiopasdfghjklñzxcvbnm 0123456789.QWERTYUIOPZXCVBNMLKJHGFDSA-".split("")
const key = "01m2n3b4v5c6x7z8a9sdf.ghjklñpoiuytr ewqASDFGHJKLÑPOIUYTREWQMNBV-CXZ".split("")



function encode(text) {
    var encodeText = ''
    var textArray = text.split("").reverse()

    textArray.map((i) => {
        encodeText += key[payloads.indexOf(i)]
    })
    console.log(encodeText)
    return encodeText
}


encode("Mi nombre es Camila")

File: auth/utils/encoder.py


payloads = "qwertyuiopasdfghjklñzxcvbnm 0123456789.QWERTYUIOPZXCVBNMLKJHGFDSA-"
key = "01m2n3b4v5c6x7z8a9sdf.ghjklñpoiuytr ewqASDFGHJKLÑPOIUYTREWQMNBV-CXZ"


def decode(messege:str) -> (str):
    
    messegeDecode = ''

    for char in messege[::-1]:
        messegeDecode += payloads[key.find(char)]

    return messegeDecode


def encode(messege:str) -> (str):
    messege = messege[::-1]
    encodeMessege = ''

    for char in messege:
        encodeMessege += key[payloads.find(char)]


    return encodeMessege



